package.json

```diff
@@ -1,27 +1,27 @@
 {
-  "name": "url",
+  "name": "@zekkkii/url",
```

Comment:
Package name should be unique if it is not marked as private. Otherwise, you scope your package with your username.